drodrpg/DRODLib/Character.cpp

```diff
@@ -3200,6 +3209,7 @@ void CCharacter::Process(
 			break;
 
 			case CCharacterCommand::CC_LevelEntrance:
+				if (bRoomBeingDisplayedOnly) return;
```

Comment:
It is useful to place the `return` on a new line (indented, of course), so that when running in the debugger, one can place a breakpoint on the `return` to easily catch when it happens.

Reply:
I left this one as a return, but put it on a new line. We never want to label jump if this one fails.